common/types/null.go

```diff
@@ -38,16 +40,18 @@ func (n Null) ConvertToNative(typeDesc reflect.Type) (interface{}, error) {
 	switch typeDesc.Kind() {
 	case reflect.Ptr:
 		switch typeDesc {
-		case jsonValueType:
-			return &structpb.Value{
-				Kind: &structpb.Value_NullValue{
-					NullValue: structpb.NullValue_NULL_VALUE}}, nil
 		case anyValueType:
 			pb, err := n.ConvertToNative(jsonValueType)
 			if err != nil {
 				return nil, err
 			}
 			return ptypes.MarshalAny(pb.(proto.Message))
+		case jsonValueType:
```

Comment:
Should also have a case for g.p.NullValue.

Reply:
I'll add a test case for this. It probably needs to sit in the case `reflect.Int32`